packages/protocol/runTests.js

```diff
@@ -17,7 +17,7 @@ async function startGanache() {
     network_id: network.network_id,
     mnemonic: network.mnemonic,
     gasPrice: network.gasPrice,
-    gasLimit: 7000000,
+    gasLimit: 8000000,
```

Comment:
You will also have to increase the limit in scripts/bash/ganache.sh and scripts/devchain.ts. I think the latter should fix contractkit tests failing.